applications/FluidDynamicsApplication/custom_utilities/estimate_dt_utilities.h

```diff
@@ -42,13 +40,22 @@ namespace Kratos
 ///@{
 
 /// Estimate the time step in a fluid problem to obtain a given Courant number.
-template< unsigned int TDim >
-class EstimateDtUtility
+class KRATOS_API(FLUID_DYNAMICS_APPLICATION) EstimateDtUtility
 {
 public:
 
-    ///@name Life Cycle
-    ///@{
+	///@name Type Definitions
+	///@{
+
+	/// Pointer definition of EstimateDtUtility
+	KRATOS_CLASS_POINTER_DEFINITION(EstimateDtUtility);
+
+    /// Function type for the element size calculator function
+    typedef std::function<double(const Geometry<Node<3>>&)> ElementSizeFunctionType;
```

Comment:
seems to be unused in the header, can be moved to the Cpp

Reply:
Same comment as for the static functions declaration, I like to declare all types in the header. IMO this simplifies going through the code.